PlastomeRegionBurstAndAlign.py

```diff
@@ -340,165 +322,154 @@ def main(args):
         if select_mode == 'int':
             ExtractAndCollect(main_d_nucl).do_INT(main_d_intron2, rec, log)
             main_d_nucl.update(main_d_intron2)
-        # Check if main dictionary empty
+
         if not main_d_nucl.items():
-            log.critical("No items in main dictionary" % out_dir)
```

Comment:
Reminding the user of the main directory can be helpful. Thus, please change to (or similar):

    log.critical("No items in main dictionary: %s" % out_dir)